hubapi/rapid-scans-api.go

```diff
@@ -51,34 +52,63 @@ type ComponentLicense struct {
 	Meta       Meta   `json:"_meta"`
 }
 type Risk struct {
-	Critical int  `json:"critical"`
-	High     int  `json:"high"`
-	Medimum  int  `json:"medium"`
-	Low      int  `json:"low"`
-	Unscored int  `json:"unscored"`
+	Critical int `json:"critical"`
+	High     int `json:"high"`
+	Medium   int `json:"medium"`
```

Comment:
Spelling is good, but this introduces a backward compatibility issue...

Reply:
I have reverted that change. We can keep the typo